Kunena/Kunena-Forum
# [K6.0] Blocked direct images pastes which result in data:image garbage in the message text

Pull Request for Issue #9475 
 
#### Summary of Changes 
Added a small plugin to CKEditor that disallow direct pasting of image data to prevent garbage.
 
#### Testing Instructions
See issue #9475 - now when trying to paste such an image a message will be shown and the paste will be blocked.
